frontend/src/views/PrivacyPolicy.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div class="container" v-if="!privacyPolicyDetails.CUSTOM_PP_DOCUMENT">
+    <h1>{{ productName }} Privacy Policy</h1>
+
+    <p>Last updated: {{ lastUpdated }}</p>
+    
+    <p>
+    <h2>Definitions of Roles and Terminology</h2>
```

Comment:
General point throughout this file - shouldn't the headings be before the `<p>` tag rather than within the paragraph?